practice-app/client/pages/_app.js

```diff
@@ -22,7 +31,9 @@ function MyApp({ Component, pageProps }) {
                 if (token) {
                     config.headers.Authorization = `Bearer ${token}`;
                 }
-                setLoading(true);
+                if(!config.url.includes("/chess/make_move")) {
```

Comment:
I understood what you are doing here; however, adding specific endpoints to the global interceptor one by one may lead to unexpected scenariois. We can pass parameters to the axios interceptor but not to the server https://stackoverflow.com/a/70647045/12277831

Reply:
Added DISABLE_LOADING: true to config. checking value of DISABLE_LOADING in _app.js